AzureCalling/Controllers/InviteViewController.swift

```diff
@@ -107,11 +114,27 @@ class InviteViewController: UIViewController {
                                               text: groupCallId ?? "",
                                               iconImage: image)]
 
-        let activityVC = UIActivityViewController(activityItems: objectsToShare,
+        shareSheetActivityVC = UIActivityViewController(activityItems: objectsToShare,
                                                   applicationActivities: nil)
+
+        guard let activityVC = shareSheetActivityVC else {
+            return
+        }
+        // On iPad the UIActivityViewController will be displayed as a popover.
+        // It requires to set a non-nil sourceView to specify the anchor location for the popover.
+        if let popoverPresentationController = activityVC.popoverPresentationController {
+            popoverPresentationController.sourceView = self.view
+            setPopoverSourceView(for: popoverPresentationController)
```

Comment:
And this can be replaced with:
`popoverPresentationController.canOverlapSourceViewRect = true`